auction9-backend/serverless.yml

```diff
@@ -0,0 +1,32 @@
+# NOTE: update this with your service name
+service: auction9-backend
+
+# Create an optimized package for our functions 
+package:
+  individually: true
+
+plugins:
+  - serverless-bundle # Package our functions with Webpack
+  - serverless-offline
+  - serverless-dotenv-plugin # Load .env as environment variables
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: dev
+  region: us-east-1
```

Comment:
Let's put 'eu-west-1' as default region. We usually use that one because it supports most of aws services such as us-east-1, but it's closer to us.

Reply:
Region change to 'us-west-1'

Reply:
Little mistake, eu instead of us.

Reply:
Fixed